src/Compiler/Interactive/fsi.fs

```diff
@@ -1342,6 +1342,20 @@ let scriptingDirectory =
 
     createDirectory (Path.Combine(Path.GetTempPath(), $"{DateTime.Now:s}-{Guid.NewGuid():n}".Replace(':', '-')))
 
+let deleteScripts () =
+    try
+#if !DEBUG
+            if scriptingDirectory.IsValueCreated then
+                if Directory.Exists(scriptingDirectory.Value) then
+                    Directory.Delete(scriptingDirectory.Value, true)
+#else
+            ()
+#endif
+    with _ ->
+        ()
+
+do AppDomain.CurrentDomain.ProcessExit |> Event.add (fun _ -> deleteScripts ())
+
```

Comment:
Oh, that's cursed

Reply:
You know the name is wrong it should probably be deleteScriptsSymbols now, because the only thing in it is .pdb files.